### DISCUSSION TITLE
Syntax highlighting override

### DISCUSSION THREAD

**Discussion by blackat** [mmistakes/minimal-mistakes#2278] [Q&A] [answered]
Hello,
I  have tried to override the $basexx variables but it seems they are not included in the main.css file.
I have put them in the assets/css/main.scss before the imports but it does not work. If I put $link-color it works in the sense that I can override the color of the links.

Am I doing something wrong? Any hint?

Thanks a lot for your time.

**Comment by mmistakes** [comment, marked as answer]
I just tested creating `/assets/css/main.scss` and added the base16 variables with different colors before the other Sass imports and the colors changed for me.

Example:

```scss
---
# Only the main Sass file needs front matter (the dashes are enough)
---

@charset "utf-8";

$base00: #1d1f21;
$base01: #282a2e;
$base02: #373b41;
$base03: #969896;
$base04: #b4b7b4;
$base05: #c5c8c6;
$base06: #e0e0e0;
$base07: #ffffff;
$base08: #cc6666;
$base09: #de935f;
$base0a: #f0c674;
$base0b: #b5bd68;
$base0c: #8abeb7;
$base0d: #81a2be;
$base0e: #b294bb;
$base0f: #a3685a;

@import "minimal-mistakes/skins/{{ site.minimal_mistakes_skin | default: 'default' }}"; // skin
@import "minimal-mistakes"; // main partials
```

**Comment by blackat** [comment]
I will try again then.
Thanks for the reply.

**Comment by blackat** [comment]
My error, sorry, it works like a charm.

**Comment by ElectricRCAircraftGuy** [comment]
I'd like to obtain a syntax highlighting identical to either GitHub (as we see if I put code blocks right here) or Stack Overflow. How can I identify exactly what color values to set for each of my `$base` settings above to achieve this?

**Comment by mmistakes** [comment]
@ElectricRCAircraftGuy The colors map to the styling in this Sass partial: https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_syntax.scss

You'll have to determine how they map, but there are plenty of base16 themes out in the wild for you to reference and drop in.

**Comment by mmistakes** [comment]
https://github.com/mmistakes/minimal-mistakes/issues/2770#issuecomment-752800124

**Comment by alichtman** [comment]
I set the syntax highlighting scheme to gruvbox for the theme I'm using (`contrast`) in https://github.com/alichtman/alichtman.github.io/commit/7c1a33636bca30f3717f1f0ed3a1ab80409d19c8
